source/utilities.cc

```diff
@@ -21,6 +21,13 @@
 #include <aspect/utilities.h>
 #include <aspect/simulator_access.h>
 
+#ifdef HAVE_LIBDAP
+#include <D4Connect.h>
```

Comment:
I am surprised that libdap does not put its header files into a folder. A header like Array.h has the potential for conflicts.